### DISCUSSION TITLE
Is there any hosted Chat App to get an Idea of how it works.

### DISCUSSION THREAD

**Discussion by naseeb0** [microsoft/AI-Chat-App-Hack#37] [General]
Is there any hosted Chat App to get an Idea of how it works. We have been seeing the local host on our sessions. If we could try out and experiement with it then may be help us someway to get an idea for hacakthon.
Thanks 
@Pamela Fox if you could help here it would be great

**Comment by pamelafox** [comment]
Sure, here's one that's currently deployed: https://app-backend-j25rgqsibtmlo.azurewebsites.net/
